File: HAI811I-ProgrammationMobile/TP2/TP2/app/src/main/java/fr/umontpellier/tp2/MainActivity.java
package fr.umontpellier.tp2;

import androidx.appcompat.app.AppCompatActivity;

import android.hardware.Sensor;
import android.hardware.SensorManager;
import android.os.Bundle;
import android.widget.ListView;

import java.util.List;

public class MainActivity extends AppCompatActivity
{
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        this.setContentView(R.layout.activity_main);

        this.init();
    }

    private void init()
    {
        ListView sensors = this.findViewById(R.id.sensors);
        sensors.setAdapter(new SensorTypeAdapter(SensorTypes.getAll()));
    }
}

File: Semestre2/HAI811I-ProgrammationMobile/CarHiboux/app/src/main/java/fr/umontpellier/carhiboux/activity/MainActivity.kt
package fr.umontpellier.carhiboux.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import fr.umontpellier.carhiboux.R

class MainActivity : AppCompatActivity()
{
    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)
    }
}